libseccomp-sys/build.rs

```diff
@@ -14,9 +14,18 @@ fn main() {
     }
 
     let link_type = match env::var("LIBSECCOMP_LINK_TYPE") {
-        Ok(v) => v, // static, framework, dylib
+        Ok(link_type) => {
+            if link_type == "framework" {
+                return Err("Seccomp is a Linux specific technology".into());
+            } else {
+                // static or dylib
+                link_type
+            }
+        }
```

Comment:
I don't really like the nesting (if-in-match) maybe we can 

```suggestion
        Ok(link_type) if link_type == "framework" => return Err("Seccomp is a Linux specific technology".into()),
        Ok(v) => v, // static or dylib
```

or

```suggestion
        // Probably requires additional let binding and .as_deref()
        Ok("framework") => return Err("Seccomp is a Linux specific technology".into()),
        Ok(v) => v, // static or dylib
```

Reply:
Thank you for the review. Sure, I'll fix it.